<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eyestack::Com_MVS::MvsCameraConfigUi</class>
 <widget class="QWidget" name="Eyestack::Com_MVS::MvsCameraConfigUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>586</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="_3">
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="_4">
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>刷新</string>
          </property>
          <property name="icon">
           <iconset resource="../Asset/asset.qrc">
            <normaloff>:/icon/fa/sync-alt-solid.svg</normaloff>:/icon/fa/sync-alt-solid.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="chooseButton">
          <property name="toolTip">
           <string>显示正在使用相机的画面</string>
          </property>
          <property name="text">
           <string>应用选中</string>
          </property>
          <property name="icon">
           <iconset resource="../Asset/asset.qrc">
            <normaloff>:/icon/fa/check-double-solid.svg</normaloff>:/icon/fa/check-double-solid.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="camlist"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="configTab">
      <attribute name="title">
       <string>配置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="impropButton">
           <property name="text">
            <string>导入属性树...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="expropButton">
           <property name="text">
            <string>导出属性树...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ContinueButton">
           <property name="text">
            <string>连续模式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="TriggerButton">
           <property name="text">
            <string>触发模式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="SoftTrigger">
           <property name="text">
            <string>软触发</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SoftTriggerOnce">
           <property name="text">
            <string>软触发一次</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="SaveJPGButton">
           <property name="text">
            <string>保存JPG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SaveBMPButton">
           <property name="text">
            <string>保存BMP</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>曝光时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ExposureTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="GetExposureTime">
           <property name="text">
            <string>参数获取</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SetExposureTime">
           <property name="text">
            <string>参数设置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Design::Monitor" name="configMonitor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>500</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="runningTab">
      <attribute name="title">
       <string>监控</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="_2">
         <item>
          <widget class="QRadioButton" name="grayscaleRadio">
           <property name="text">
            <string>灰度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="coloredRadio">
           <property name="text">
            <string>彩色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>取帧时限：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="timeoutSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Design::Monitor" name="runningMonitor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Design::Monitor</class>
   <extends>QWidget</extends>
   <header>Eyestack/Design/Monitor.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Asset/asset.qrc"/>
 </resources>
 <connections/>
</ui>
